### ISSUE TITLE
Different accessibility of polls

### ISSUE THREAD

**Issue by stanriders** [stanriders/circleforms#6]
- [ ] Poll types
  - [x] Public poll (can be accessed directly)
  - [x] Link-only poll (poll link should have a key for accessing it)
  - [ ] _(low-prio)_ Poll limited to a set of users (won't be accessible unless user is in the set, pretty much invites)
  - [ ] _(low-prio)_ Friends-only poll? (friend list can be taken from API)

* [ ] Poll results accessibility  
  * [ ] Always public
  * [ ] Public after completion
  * [ ] Link only
  * [ ] Invites
  * [ ] Host only
* [x] Rank/pp/gamemode limited polls  
  * [x] Rank/pp
  * [x] Gamemode only
* [ ] _(low-prio)_ User blacklisting
